src/integrationTest/resources/application-int-test.yaml

```diff
@@ -12,6 +12,7 @@ darts-gateway:
     map-token-to-session: false
     share-token-for-same-credentials: false
     entry-time-to-idle-seconds: 1
+    share-token-allocation-early-expiration-minutes: 300
```

Comment:
Wondering if this is too long - 5 hours?
Only because I'm sure our current access_tokens don't live for very long, certainly less than 3 hours
@hemantasharma1129

Reply:
haha yep my bad should be 5 minutes for this got my figures the wrong way around